source/Ship.cpp

```diff
@@ -3324,13 +3325,25 @@ double Ship::Health() const
 // Get the hull fraction at which this ship is disabled.
 double Ship::DisabledHull() const
 {
-	double hull = attributes.Get("hull");
+	double hull = MaxHull();
 	double minimumHull = MinimumHull();
 
 	return (hull > 0. ? minimumHull / hull : 0.);
 }
 
 
+// Get the maximum shield and hull values of the ship, accounting for multipliers.
+double Ship::MaxShields() const
+{
+	return attributes.Get("hull") * (1 + attributes.Get("shield multiplier"));
```

Comment:
As mentioned in the comments, I prefer a multiplier to be a true multiplier.

Reply:
I definitely agree with Petervdmeer on this. It is both easier to understand and simpler to have the multiplier be the actual multiplier.